include/intx/int128.hpp

```diff
@@ -358,9 +358,23 @@ constexpr uint128 operator>>(uint128 x, uint128 shift) noexcept
 /// Multiplication
 /// @{
 
-/// Portable full unsigned multiplication 64 x 64 -> 128.
-constexpr uint128 constexpr_umul(uint64_t x, uint64_t y) noexcept
+
+/// Full unsigned multiplication 64 x 64 -> 128.
+inline constexpr uint128 umul(uint64_t x, uint64_t y) noexcept
 {
+#if defined(__SIZEOF_INT128__)
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wpedantic"
+    if (!is_constant_evaluated())
```

Comment:
Using builtin uint128 is fine in constexpr too. The `is_constant_evaluated()` is only actually needed for MSVC usage of intrinsic. We can keep it if not too big problem.